modules/thermal_hydraulics/include/materials/ADWallHeatTransferCoefficientKazimiMaterial.h

```diff
@@ -0,0 +1,48 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Material.h"
+
+/**
+ *  Computes wall heat transfer coefficient for liquid sodium using Kazimi-Carelli correlation
+ */
+class ADWallHeatTransferCoefficientKazimiMaterial : public Material
+{
+public:
+  ADWallHeatTransferCoefficientKazimiMaterial(const InputParameters & parameters);
+
+protected:
+  virtual void computeQpProperties();
```

Comment:
```suggestion
  virtual void computeQpProperties() override;
```

Reply:
Apply this change to the other materials as well.

Reply:
done